performant-software/Annotation-Studio
# For documents with no assigned users, don't show the user select list

## What does this PR do?
1. For documents with no assigned users, don't show the user select list
2. For documents with no assigned users, don't show the switch user list

## How to test?
1. Sign in as an admin
2. Navigate to http://cove-staging.herokuapp.com/documents/alice-looking-glass
3. There should not be a by user or a switch user dropdown
